Configurations/unix-Makefile.tmpl

```diff
@@ -686,8 +686,10 @@ tar:
 	DISTDIR=$(NAME); \
 	mkdir -p $$TMPDIR/$$DISTDIR; \
 	(cd $(SRCDIR); \
+	 submodule_re=`git submodule status | sed -e 's/^.//' | cut -d' ' -f2`; \
+	 submodule_re="^(`echo $$submodule_re | sed -e 's/ /|/g'`)\$$"; \
 	 git ls-tree -r --name-only --full-tree HEAD \
-         | grep -v '^fuzz/corpora' \
+	 | grep -v '^fuzz/corpora' | egrep -v "$$submodule_re" \
```

Comment:
The submodule regexp is one ending in a `$`, while the fuzz corpora one isn't.  Of course, I could make the regexp end up like this:

```
^(boringssl$|krb5$|pyca-cryptography$|fuzz/corpora)
```

but that makes for a slightly more complex generation of the regexp, and I didn't find that worth the while.

Regard egrep vs -E, how far back did -E exist?  Will it work on all platforms we support?